docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-113.adoc

```diff
@@ -0,0 +1,47 @@
+== MonkeyLearn API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_113
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+A MonkeyLearn API Key is essential for accessing the MonkeyLearn text analysis services, which provide powerful machine learning models for text data. The key is critical for authenticating requests to the API and ensuring that only authorized applications can utilize the service. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data leaks. It is vital to manage this key securely and regularly review your security practices to prevent any unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://monkeylearn.com/api/v3/#authentication[MonkeyLearn documentation and support] resources.
```

Comment:
The MonkeyLearn API Key serves as an essential credential for accessing MonkeyLearn's machine learning models for text data analysis. This key acts as a security measure, guaranteeing that only authorized applications can interact with the API and utilize the services provided. Exposing the API key can lead to unauthorized access, misuse of the API, and potential data leaks. For this reason, it's crucial to manage your MonkeyLearn API Key securely and regularly review your security practices to prevent any unauthorized usage.

For comprehensive security guidelines and best practices regarding MonkeyLearn API Key management, refer to the latest documentation and support resources available at [Link to MonkeyLearn documentation and support](https://monkeylearn.com/api/v3/#authentication).